src/pages/index.js

```diff
@@ -132,3 +218,23 @@ profileEditBtn.addEventListener("click", () => {
 addNewCardBtn.addEventListener("click", () => {
   addCardPopup.open();
 });
+
+deleteButton.addEventListener("click", () => {
```

Comment:
Please, delete it

`PopupConfirmation` should add the listener for deleting

Reply:
Hi @gennady-bars, many thanks for your feedback. Really appreciate it. Can you provide further clarity on what you would like me to delete here. Apologies as I'm not really sure.